finished styling admin view

AntonioPavkovic opened the pull request.

AntonioPavkovic committed:
finished styling admin view

jozogs reviewed (changes requested):
Have found some errors, pointed @AntonioPavkovic to them in our PM

AntonioPavkovic committed:
changed blood types form _NEGATIVE/_POSITIVE to +/-
changed text field color to red
add IgnorePointer to name, date and blood_type fields in donation form
